Sources/PrimerSDK/Classes/User Interface/Text Fields/PrimerCVVFieldView.swift

```diff
@@ -45,15 +45,22 @@ public final class PrimerCVVFieldView: PrimerTextFieldView {
             validation = .notAvailable
         }
         
+        primerTextField._text = newText
+        primerTextField.text = newText
+        
         switch validation {
         case .valid:
             delegate?.primerTextFieldView(self, isValid: true)
+        case .invalid:
+            if let cvvLength = cardNetwork.validation?.code.length, newText.count == cvvLength {
+                delegate?.primerTextFieldView(self, isValid: false)
+            } else {
+                delegate?.primerTextFieldView(self, isValid: nil)
```

Comment:
[Q]
Just so that I understand `isValid` - is that correct?
* `false` means "it's not valid"
* `nil` means "we don't now yet"
* `true` means "it's valid"

Reply:
yes